SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialComponentTest/SpatialComponentTestCallbackComponent.cpp

```diff
@@ -40,11 +40,9 @@ void USpatialComponentTestCallbackComponent::AddAndRemoveComponents()
 {
 	// Add 2 components
 	USpatialComponentTestDummyComponent* TestComponent1 = NewObject<USpatialComponentTestDummyComponent>(this);
-	TestComponent1->AttachToComponent(this, FAttachmentTransformRules::KeepRelativeTransform);
```

Comment:
Doesn't seem to be needed and causes errors in native, since this puts it into the attachment replication, and the dummy component is not supported for networking.